resources/requirements_mypy.txt

```diff
@@ -0,0 +1,3 @@
+mypy==0.982
+types-PyYAML==6.0.12.1
+types-setuptools==65.5.0.2
```

Comment:
I see benefits from pin `mypy` but I'm not sure if two others also should be pinned. On one side it reduces the probability of random error, on the other side, I do not remember errors with this packages. Maybe release this versions to reduce dependaboot PRs?

Reply:
I agree and vote to pin them. There is a strong advantage to having consistent type checking, so it is nice to make sure these packages don't change unexpectedly.

Reply:
Yeah I agree, I don't mind a few extra PRs if it means we don't get spurious unexpected failures.